src/app/store.ts

```diff
@@ -1,25 +1,24 @@
 import * as _ from 'lodash'
-import { applyMiddleware, combineReducers, createStore, Store } from 'redux'
+import { applyMiddleware, combineReducers, createStore, Middleware, Reducer } from 'redux'
 import logger from 'redux-logger'
-import { combineEpics, createEpicMiddleware } from 'redux-observable'
+import { combineEpics, createEpicMiddleware, Epic } from 'redux-observable'
+import thunk from 'redux-thunk'
+
 import epics from './epics'
 import * as reducers from './reducers'
+import { StoreState } from './types'
 
-const appReducer: any = combineReducers({ ...reducers })
-const appEpics: any = combineEpics(..._.values(epics))
-
-const epicMiddleWare: any = createEpicMiddleware(appEpics)
-
-const reduxMiddleWares: any = []
+const appReducer: Reducer<StoreState> = combineReducers({ ...reducers })
+const appEpics: Epic<any, any, any, any> = combineEpics(..._.values(epics))
 
+const reduxMiddleWares: Middleware[] = []
 if (process.env.NODE_ENV !== 'production') {
   reduxMiddleWares.push(logger)
 }
-reduxMiddleWares.push(epicMiddleWare)
+reduxMiddleWares.push(createEpicMiddleware(appEpics))
+reduxMiddleWares.push(thunk)
```

Comment:
Thunk was added for the unlocking actions but now that we are using observables we shouldn't need it anymore.

Reply:
Ok, I will remove it.